USAGE.md

```diff
@@ -0,0 +1,33 @@
+# How to use Testing Farm as GitHub Action in your projects
+
+Before start of using Testing Farm as GitHub Action in your projects,
+there are several steps which you have to do it.
+
+## As Testing Farm team for public or private ranch
+
+Before any implementation as Testing Farm team for API_KEY here [onboarding](https://docs.testing-farm.io/general/0.1/onboarding.html).
```

Comment:
s/As/Ask/ in the headline and also in the paragraph. It might be good to explain the difference between public and private ranch here, when it's used here.

Reply:
Good point.

Reply:
Fixed by https://github.com/sclorg/testing-farm-as-github-action/pull/64/commits/cf1e8e968831ae6d1a95284a5838c2c152d46185 and https://github.com/sclorg/testing-farm-as-github-action/pull/64/commits/d3ab80538320df2b1fb56b74361edad681d00dd7

Reply:
This does not seem to be fixed yet.

Reply:
See here: https://github.com/sclorg/testing-farm-as-github-action/pull/64/commits/d3ab80538320df2b1fb56b74361edad681d00dd7 at the beginning.

Reply:
I have rebased the pull request.

Reply:
https://github.com/sclorg/testing-farm-as-github-action/pull/64/files#diff-58a825c70bccfbe691d7ddbddb133275f4f657de6aeb4a502fa9fce279f8bf8dR9

Reply:
I do not see it here.